ios-trivia-game/ViewControllers/GameOptionsViewController.swift

```diff
@@ -33,3 +78,30 @@ class GameOptionsViewController: UIViewController {
     */
 
 }
+
+extension GameOptionsViewController: UIPickerViewDelegate, UIPickerViewDataSource {
+    
+    // The number of columns of data
+    func numberOfComponents(in pickerView: UIPickerView) -> Int {
+        return 1
+    }
+    
+    // The number of rows of data
+    func pickerView(_ pickerView: UIPickerView, numberOfRowsInComponent component: Int) -> Int {
+        if pickerView.tag == 1 {
+            return categoryPickerData.count
+        } else {
```

Comment:
We can probably remove this else statement and return number of questions at the bottom.